build.gradle

```
@@ -81,6 +81,7 @@ apply from: 'buildSrc/buildTasks.gradle'
 apply from: 'buildSrc/runTasks.gradle'
 apply from: 'buildSrc/deploymentTasks.gradle'
 apply from: 'buildSrc/docker.gradle'
+apply from: 'buildSrc/swagger.gradle'
```

```suggestion
apply from: 'buildSrc/postman.gradle'
```
The postman.gradle file isn't applied.  Shouldn't it be applied here.  I know in a previous version shown to me the group name on the tasks was Swagger so that's why I'm wondering if it was just an oversight to update this file.